docs/data/material/discover-more/showcase/appList.js

```diff
@@ -433,18 +372,6 @@ const appList = [
     stars: 24,
     dateAdded: '2018-01-35',
   },
-  {
-    title: 'MQTT Explorer',
-    description:
-      'A comprehensive MQTT Client which visualizes broker traffic in a hierarchical view. ' +
-      'The protocol is used in many IoT and home automation scenarios, ' +
-      'making integrating new services dead easy.',
-    link: 'https://mqtt-explorer.com/',
-    source: 'https://github.com/thomasnordquist/MQTT-Explorer',
-    image: 'mqtt-explorer.png',
-    stars: 1600,
```

Comment:
1,600 stars and still active, why remove it?

Reply:
Good question. I honestly can't remember 😅 Must have been a mistake. Let's add this one back.